src/status-based-retry/README.md

```diff
@@ -0,0 +1,30 @@
+# Retry based on Rest Caller response status
+
+Using the Data Mapping task, iterate through the elements of a string array and apply transformation functions to convert each string value to uppercase.
```

Comment:
@rohitjangid - how is this different/similar to https://github.com/GoogleCloudPlatform/application-integration-samples/tree/main/src/case-conversion?

Reply:
skipped updating the description. Updated the readme